src/components/PokemonCard.js

```diff
@@ -0,0 +1,59 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+
+export default function PokemonCard({name}) {
+    const [pokemon, setPokemon] = useState({});
+
+    useEffect(() => {
+
+        async function getPokemon(name) {
+            try {
+                const {data} = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`)
+                setPokemon(data);
+            } catch (e) {
+                console.log(e)
+            }
+        }
+
+        getPokemon(name);
+
+    }, []) // <--- mount Cycle
+
+    return (
+        <>
+            {
+                pokemon.name &&
```

Comment:
`.name` is niet mogelijk als het request nog niet uitgevoerd is, dus ik zou deze aanpassen naar:
```suggestion
                pokemon &&
```